modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/css/main.scss

```diff
@@ -769,7 +769,33 @@ $productMenuWidth: 320px;
 	.upload-drop-active &,
 	.upload-drop-intent & {
 		.document-container {
-			background-color: #eef4ff;
+			background-color: $primary-l3;
+		}
+	}
+
+	.upload-drop-active & {
+		.document-container {
+			border-radius: 2px;
+			box-shadow: 0 0 0 2px $primary-l2 !important;
```

Comment:
Is this `!important` really needed?

Reply:
I removed the document-library specific from the theme [753d4c7](https://github.com/liferay-lima/liferay-portal/pull/2832/commits/753d4c78b58a6174f30c80c27125c2bd4bcfacb4) and we can drop the important.